Prox/Prox/Utilities/OpenInHelper.swift

```diff
@@ -0,0 +1,168 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+import Foundation
+import MapKit
+
+private enum Schemes: String {
+    case yelp = "www.yelp.com"
+    case tripAdvisor = "www.tripadvisor.co.uk"
+    case wikipedia = "en.wikipedia.org"
+}
+
+struct OpenInHelper {
```

Comment:
Structs are far more lightweight than classes, event when completely empty they take less space and memory. And having no state and being essentially immutable struct is a good way of producing helper objects like this. A class would be overkill.